### DISCUSSION TITLE
Should I export the source or build of the packages in a private monorepo?

### DISCUSSION THREAD

**Discussion by amir-ziaei** [vercel/turborepo#8783] [Help] [answered]
### Summary

Hello!

I have a completely private monorepo containing several applications and packages. None of these packages will ever need to be published to NPM.

Given this context, I'm wondering whether it makes sense for my packages to export their build/dist versions, or if it's sufficient/better to export the source files instead. I am leaning towards the latter because I can't think of a good reason to keep constantly building these packages and consuming their built versions.

Any advice or insights would be greatly appreciated! Thank you!

**Comment by anthonyshew** [comment, marked as answer]
We have documentation on the tradeoffs written here: https://turbo.build/repo/docs/core-concepts/internal-packages#compilation-strategies

All depends on what you want to do for your case!
